etc/omero.properties

```diff
@@ -915,10 +915,15 @@ Ice.IPv6=1
 #############################################
 product.name=OMERO.server
 
+
 #############################################
 ## Library versions
 #############################################
+
+org.bioformats=ome
```

Comment:
Is this block separating our maintained dependencies from external dependencies?

Reply:
That was my feeling. It "felt wrong" to have an unalphabetized section:

```
...
org.bioformats=
versions.bioformats
...
```

but it also felt wrong not to have them together. When I thought about having a maintained section, then it started getting complicated (omero.slice, etc) so I tried to keep it simple.

Reply:
I had the same feeling while thinking at the list and the maintained section.  As this is largely internal and the alphabetical order is not strictly respected, I'd say either integrate as this is or putting these 2 lines consecutively into the main list are completely acceptable choices for the time being.